lib/view.js

```diff
@@ -0,0 +1,56 @@
+// Copyright IBM Corp. 2015,2016. All Rights Reserved.
```

Comment:
Does the copyright tool make those changes for us, now? We can always run a separate PR to update it.